internal/discord/command-handlers.go

```diff
@@ -28,8 +28,10 @@ var (
 // AddCommands registers the slash commands with Discord
 func (d *Discord) AddCommands(s *discordgo.Session, event *discordgo.Ready) {
 	fmt.Printf("Initializing Discord...\n")
-
 	for _, discordGuild := range event.Guilds {
+		if discordGuild.ID != d.GuildID {
+			continue
```

Comment:
ah i missed the forloop above. hmm i'm thinking we would still want to error if we can't find ANY matching guild ids